backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/migration/MigrationTask.java

```diff
@@ -140,6 +140,10 @@ public String type() {
         return CASSANDRA_MIGRATION;
     }
 
+    SchemaVersion getTargetVersion() {
```

Comment:
Should not it be in the first PR?

Reply:
I don't think so, it was not used (otherwise it would not compile)